keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -457,11 +457,12 @@ def _track_regularization(self):
 
     def train_step(self, data):
         x, y = data
-        y_for_metrics, y_training_target = y
+        gt_boxes = y["gt_boxes"]
+        gt_classes = y["gt_classes"]
 
         with tf.GradientTape() as tape:
             y_pred = self(x, training=True)
-            loss = self._backward(y_training_target, y_pred)
+            loss = self._backward(gt_boxes, gt_classes, y_pred)
```

Comment:
should we just entirely remove the metrics now?   We CANT evaluate them if we don't pass the raw boxes forward.

Reply:
Done